src/mono/browser/runtime/startup.ts

```diff
@@ -617,7 +617,41 @@ export function mono_wasm_load_runtime (): void {
         if (!loaderHelpers.isDebuggingSupported() || !runtimeHelpers.config.resources!.pdb) {
             debugLevel = 0;
         }
-        cwraps.mono_wasm_load_runtime(debugLevel);
+
+        const runtimeConfigProperties = new Map<string, string>();
+        if (runtimeHelpers.config.runtimeConfig?.runtimeOptions?.configProperties) {
+            for (const [key, value] of Object.entries(runtimeHelpers.config.runtimeConfig?.runtimeOptions?.configProperties)) {
+                runtimeConfigProperties.set(key, "" + value);
+            }
+        }
+        runtimeConfigProperties.set("APP_CONTEXT_BASE_DIRECTORY", "/");
+        runtimeConfigProperties.set("RUNTIME_IDENTIFIER", "browser-wasm");
+        const propertyCount = runtimeConfigProperties.size;
+
+        const buffers:VoidPtr[] = [];
+        const appctx_keys = malloc(4 * runtimeConfigProperties.size) as any as CharPtrPtr;
+        const appctx_values = malloc(4 * runtimeConfigProperties.size) as any as CharPtrPtr;
```

Comment:
It's the `4` here because of 32 bit addressing?

Reply:
yes